XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -416,9 +422,6 @@
     <Compile Include="ControlPages\InkCanvasPage.xaml.cs">
       <DependentUpon>InkCanvasPage.xaml</DependentUpon>
     </Compile>
-    <Compile Include="ControlPages\InkToolbarPage.xaml.cs">
```

Comment:
? Please revert this change.

Reply:
Addded back in upcoming commit. I probably accidentally unloaded from project when i was trying to do so for the ItemsRepeater page.